packages/devextreme/js/ui/drop_down_box.js

```diff
@@ -206,33 +198,25 @@ const DropDownBox = DropDownEditor.inherit({
         return deferred.promise();
     },
 
-    _popupElementTabHandler: function(e) {
+    _popupTabHandler: function(e) {
         if(normalizeKeyName(e) !== 'tab') return;
 
         const $firstTabbable = this._getTabbableElements().first().get(0);
         const $lastTabbable = this._getTabbableElements().last().get(0);
-        const $target = e.originalEvent.target;
-        const moveBackward = !!($target === $firstTabbable && e.shift);
-        const moveForward = !!($target === $lastTabbable && !e.shift);
+        const $target = e.target;
```

Comment:
Why do we work with `e` instead of `e.originalTarget` now?

Reply:
because now it handles keydown event directly instead of registerKeyHandler